specification/compute/resource-manager/Microsoft.Compute/stable/2021-11-01/examples/compute/GetVirtualMachine.json

```diff
@@ -25,7 +25,11 @@
             "id": "/subscriptions/{subscription-id}/resourceGroups/myResourceGroup/providers/Microsoft.Compute/proximityPlacementGroups/my-ppg01"
           },
           "hardwareProfile": {
-            "vmSize": "Standard_DS3_v2"
+            "vmSize": "Standard_DS3_v2",
+            "vmSizeProperties": {
+              "vCPUsAvailable": 1,
+              "vCPUsPerCore": 1
+            }
```

Comment:
We should add a separate example file for GetVirtualMachineWithVMSizeProperties as vmSizeProperties is optional and is not part of the basic VM api response unless the user created the VM with vmSizeProperties.